<launch>
    <node  respawn="true" pkg="my_serial" type="robot_control" name="robot_control" output="screen">
       <rosparam file="$(find my_serial)/param/AGV_base.yaml"/>
      
    </node>
</launch>
